src/auth/bearer/fake/fake.cc

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2014, Treehouse Networks Ltd. New Zealand
```

Comment:
The fake helpers are used as example code and dual-licensed to encourage people using the displayed logic as a basis for their own non-GPL helpers instead of reinventing the helper protocol I/O portions with various mistakes.
The issue here is that the standard boilerplate part is missing on a number of the new files. Will add that ASAP.